packages/client/css/glsp-sprotty.css

```diff
@@ -230,6 +230,10 @@
     opacity: 1;
 }
 
+.sprotty.sprotty-hidden .sprotty-projection-bar {
+    visibility: hidden;
```

Comment:
What is the reason we need to hide them explicitly?

Reply:
The projection bars are still visible in the hidden div which leads to a weird render artifact in the top right corner of the standalone example.
I guess the reason for this, that the "hidden div" is not actually hidden its just a div with 0 width and height.

![image](https://github.com/eclipse-glsp/glsp-client/assets/2311075/0c19d501-25e2-4e5b-a5ce-b409b598651c)

Reply:
Ah, good catch! I didn't notice that one before but with your fix it is gone.